test/compilable/interpret3.d

```diff
@@ -1,4 +1,4 @@
-﻿// PERMUTE_ARGS: -inline
+// PERMUTE_ARGS: -inline
```

Comment:
What's going on with this?

Reply:
Trailing whitespace - my editor automatically removes it. Sorry